src/manifest.json

```diff
@@ -49,22 +49,42 @@
         "js/firstparties/twitter.js"
       ],
       "matches": [
-        "https://twitter.com/*",
-        "http://twitter.com/*"
+        "*://twitter.com/*"
```

Comment:
Let's revert and go back to explicitly listing out https and http. "*://" is confusing ... it [includes ws(s) in some browsers but not others](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns#Browser_compatibility).